src/featureManagement/FeatureFlagOptions.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { FeatureFlagRefreshOptions } from "../RefreshOptions";
+import { SettingSelector } from "../types";
+
+/**
+ * Options used to configure feature flags.
+ */
+export interface FeatureFlagOptions {
+    /**
+     * Specifies whether feature flag support is enabled.
+     */
+    enabled: boolean;
+
+    /**
+     * Specifies the selectors used to filter feature flags.
+     *
+     * @remarks
+     * keyFilter of selector will be prefixed with "appconfig.featureflag/" when request is sent.
+     * If no selectors are specified then no feature flags will be retrieved.

```

Comment:
If no selectors are specified, shouldn't we load all Feature Flags with null label? That's what we do for key-values too.

Reply:
Per discussion, for feature flags we expect customers to explicitly specify selectors.

Reply:
Should we throw an error if `FeatureFlagOptions.enabled` is true but selectors are not provided?

Reply:
I agree, as it makes no sense to enable feature flags but load nothing. Throwing an error is better than silently loading nothing.

Reply:
923c124